### PR TITLE
8k5 enablement 

### PR DESCRIPTION
**Pull request by ThomasFuchs**
Signed-off-by: Thomas Fuchs <thomas.fuchs@de.ibm.com>

### CODE REVIEW

**Comment by ThomasFuchs** [comment]
Todo's that are left from 8k5_dev
### Main Steps
- [ ] 8K5 PSL Checkpoint viv17.4/psl233 generation
- [ ] 8K5 PSL Checkpoint upload to OpenPower Portal
- [ ] Hardware test ([x] hdl_example_BRAM [x] hls_memcopy
                                 [ ] hls_helloworld [ ] hls_intersect 
                                 [ ] hls_hashjoin [ ] hls_bfs [ ] hls_sponge )

### hardware setup
- [ ] `hardware/snap_check_psl` -- **TODO** Need 8K5 Checkpoint md5sum

### general
- [ ] `README.md` Announce SNAP supports a new card. **Last step**

**Comment by acastellane** [comment]
I successfully created, loaded and tested also helloworld, bfs and intersect in the 8k5 board.
Thanks

File: software/tools/snap_find_card [hunk 1]
```diff
@@ -103,6 +130,7 @@ function detect_snap_cards() {
 #
 function detect_gzip_cards() {
```
Comment (boekholt) [inline software/tools/snap_find_card]: We should not have gzip related code in the SNAP repository

**Comment by ThomasFuchs** [comment]
### 8K5_ena_rebase is now in master

**Pull request closed without merging**